client/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import "tachyons/css/tachyons.css";
+
+import Landing from "./components/Landing";
+import Navbar from "./components/Navbar";
+
+class App extends Component {
```

Comment:
there no methods from the `Component` class being used here, so this component should be a functional one